RecoLocalCalo/HGCalRecProducers/plugins/HGCalRecHitWorkerSimple.h

```diff
@@ -0,0 +1,40 @@
+#ifndef RecoLocalCalo_HGCalRecProducers_HGCalRecHitWorkerSimple_hh
+#define RecoLocalCalo_HGCalRecProducers_HGCalRecHitWorkerSimple_hh
+
+/** \class HGCalRecHitSimpleAlgo
+  *  Simple algoritm to make HGCAL rechits from HGCAL uncalibrated rechits
+  *
+  *  \author Valeri Andreev
+  */
+
+#include "RecoLocalCalo/HGCalRecProducers/interface/HGCalRecHitWorkerBaseClass.h"
+#include "RecoLocalCalo/HGCalRecAlgos/interface/HGCalRecHitSimpleAlgo.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+
+#include "FWCore/Framework/interface/ESHandle.h"
+
+class HGCalRecHitWorkerSimple : public HGCalRecHitWorkerBaseClass {
+        public:
```

Comment:
indentation is a bit excessive compared to most of the code we have
(not a big deal but still)